taskcluster/ffios_taskgraph/transforms/bitrise.py

```diff
@@ -47,25 +47,26 @@ def set_run_config(config, tasks):
 @transforms.add
 def set_worker_config(config, tasks):
     for task in tasks:
-        locale = task["locale"]
-
         worker = task.setdefault("worker", {})
         artifacts = worker.setdefault("artifacts", [])
 
-        artifacts.extend([{
+        artifacts.append({
             "type": "file",
             "name": "public/logs/bitrise.log",
             "path": "/builds/worker/checkouts/src/bitrise.log",
-        }, {
-            "type": "file",
-            "name": "public/screenshots/{}.zip".format(locale),
-            "path": "/builds/worker/checkouts/src/{}.zip".format(locale),
-        }])
+        })
+
+        for locale in task["attributes"]["chunk_locales"]:
+            artifacts.append({
+                "type": "file",
+                "name": "public/screenshots/{}.zip".format(locale),
+                "path": "/builds/worker/checkouts/src/{}.zip".format(locale),
+            })
 
         worker.setdefault("docker-image", {"in-tree": "screenshots"})
-        worker.setdefault("max-run-time", 3600)
+        worker.setdefault("max-run-time", 10800)
```

Comment:
I set the max-run-time to 3 hours. Basically, we may end up in situation where a TC worker schedules a job to bitrise, but the latter is overloaded (say because of the smoketests jobs which aren't monitored by Taskcluster). So the TC worker will schedule a job then wait for another to finish. Since a job is max 90 minutes, I set the limit to twice that value.